### DISCUSSION TITLE
Are you using those color selection for the different icons?

### DISCUSSION THREAD

**Discussion by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1011] [Polls]
A few releases ago we added the capability of selecting colors for the multiple components displayed around the various pages, however, this came with a way more complex (and slow) blueprint settings page and a bit more complexity in the code.

With that said, I'd like to understand how much are you using this functionality.

**Comment by fordcrews** [comment]
I am using them on the chips icons on the main page, the icons are so mall, I made inside ones green, outside yellow, and ones at my moms red.   

Maybe it's going to be part of the alarm support, but a page dedicated to sensors, would be nice so you could see what doors/windows are open, and where there is motion.

**Reply by edwardtfn** [reply]
> Maybe it's going to be part of the alarm support, but a page dedicated to sensors, would be nice so you could see what doors/windows are open, and where there is motion.

What about the existing entities pages?

**Reply by edwardtfn** [reply]
I understand you have a color coding for chips, but what about the first 2 icons related to the hardware relays? Are you using colors on those also?

**Reply by fordcrews** [reply]
Oops missed this, no.

**Comment by lordhelmet77** [comment]
Maybe, instead of choosing each individual icon color, it could be useful to set a color theme per page:

**Button Page 01**
- Icon Color Disabled
- Icon Color Enabled
- Button Background Disabled
- Button Background Enabled

**Comment by edwardtfn** [comment]
What about the custom color selection for time and date display?
Anyone using that?

**Reply by lordhelmet77** [reply]
Personally, I'd like the option to change the background picture down the road, custom colors would be important for that.

**Reply by edwardtfn** [reply]
This model of Nextion display used in the NSPanels does not support adding new backgrounds dynamically, so unless you are using the existing backgrounds (and those are all dark), you will have to edit the HMI anyways... so it really makes sense having the time and date colors set by the blueprint?

I will be fine if the answer is yes. The code is already implemented anyways... I just want to understand the use cases.

And I have to be honest, I'm harvesting for any byte of memory I could save, and this is one that is stored in a global var for the case the network is out.

**Reply by lordhelmet77** [reply]
In that case, 25 years of visual communication and design tell me, that white should be fine. I'd suggest that readability/contrast are key for time and date and with the current background, this is already perfect. 😊
